packages/core/src/config.ts

```diff
@@ -656,7 +656,8 @@ const composeExternalsConfig = (
     esm: 'module-import',
     cjs: 'commonjs',
     umd: 'umd',
-    mf: 'var', // same as default value
+    // if use 'var', when users use external package like '@pkg', this will cause syntax error like 'var pkg = @pkg'
```

Comment:
More than this, the common usage of browsers should not be `'var'` and not just for the sake of edge considerations.
We could change the explanation for why we choose `global` instead of `window` or other options, perhaps by referencing vmok's practice.